multi-cluster-spring-boot/README.md

```diff
@@ -28,7 +28,55 @@ This quickstart can be deployed quickly using Ansible. Here are the steps.
     ```
     $ oc login <prod cluster>
     ...
-    $ ansible-playbook -i ./applier/inventory-prod/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
+    $ ansible-playbook -i image-mirror-example/.applier/inventory-prod/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
+    ```
+5. One of the things that was created by ansible is a `ServiceAccount` that will be used for promoting your app from _Dev_ to _Prod_. We'll need to extract its credentials so that our pipeline can use that account.
+    ```
+    $ TOKEN=$(oc serviceaccounts get-token docker-registry-prod -n multicluster-spring-boot-prod)
+    ```
+    The Ansible automation for your _Dev_ cluster will expect a parameters file to be created at `./applier/params/prod-credentials`. It should look something like this:
+    ```
+    $ echo "TOKEN=${TOKEN}
+    SECRET_NAME=prod-credentials" > image-mirror-example/.applier/params/prod-credentials
+    ```
+6. We need to create the the *prod-api-credentials* param file so our pipeline will be able to verify a successful deployment to production.
+    ```
+    echo "TOKEN=${TOKEN}
+    API_URL=<API_URL>
+    REGISTRY_URL=<REGISTRY URL>
+    SECRET_NAME=prod-cluster-credentials" > image-mirror-example/.applier/params/prod-cluster-credentials
+    ```
+6. Now, Log into your _Dev_ cluster, and instantiate the pre-pipeline configuration.
+    ```
+    $ oc login <dev cluster>
+    ...
+    $ ansible-playbook -i image-mirror-example/.applier/inventory-pre-dev/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
```

Comment:
I have not found a better way to do this so far. We need to set the prod credentials in the dev cluster before running the dev setup.